mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loanaccount/LoanAccountFragment.java

```diff
@@ -337,9 +337,9 @@ private void initiateLoanCreation(LoansPayload loansPayload) {
     public void inflatesubmissionDate() {
         mfDatePicker = MFDatePicker.newInsance(this);
 
-        tv_submittedon_date.setText(MFDatePicker.getDatePickedAsString());
+        tvSubmittedOnDate.setText(MFDatePicker.getDatePickedAsString());
 
-        tv_submittedon_date.setOnClickListener(new View.OnClickListener() {
+        tvSubmittedOnDate.setOnClickListener(new View.OnClickListener() {
```

Comment:
We are using ButterKnife. Please remove this OnClickListener implementation and use Butter Knife ```@OnClick``` annotation.